experiments/mcp-eval/tasks/bigquery-1-dataset/verify.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TOP_LEVEL=$(git rev-parse --show-toplevel)
+FILE_PATH="${TOP_LEVEL}/.build/tasks/bigquery-1-dataset/dataset.yaml"
+
+# Check if the file exists
+if [[ ! -f "${FILE_PATH}" ]]; then
+  echo "File not found: ${FILE_PATH}"
+  exit 1
+fi
+
+# Check if the file is a YAML file with kind "BigQueryDataset"
+if ! grep -q "kind: BigQueryDataset" "${FILE_PATH}"; then
```

Comment:
No need to change yet, but I can imagine we can probably generalize this in future to a "required yaml", e.g. here is looks like the "required yaml" file would be:

```
kind: BigQueryDataset
spec:
  defaultTableExpirationMs: 3600000
  location: US
```

I would be interested to know how this fails also, e.g. does the model sometimes add quotes or spaces?

Reply:
Sure. but I have a generic concern about this approach. KCC is OSS. Its data could be web crawled and used to train future models. Then the tasks are contaminated. Yes, this repo already has many examples (fixtures, samples), but it's different from recording the exact eval result.